packages/react-core/src/components/Tabs/__tests__/Tabs.test.tsx

```diff
@@ -492,3 +492,29 @@ test('should not render scroll buttons when isVertical is true', () => {
   expect(screen.queryByLabelText('Scroll left')).not.toBeInTheDocument();
   expect(screen.queryByLabelText('Scroll right')).not.toBeInTheDocument();
 });
+
+test('should render a disabled add button', () => {
+  render(
+    <Tabs onAdd={jest.fn()} addButtonAriaLabel="add-label" isAddButtonDisabled={true}>
+      <Tab eventKey={0} title={<TabTitleText>"Tab item 1"</TabTitleText>} closeButtonAriaLabel="close-label">
```

Comment:
FWIW we technically shouldn't need to pass a Tab specifically for this test since we just want to check the "add" button, which lives within the Tabs component and should render regardless. So we could either just replace this Tab with a plain `<div>Tab content</div>` or remove the closeButtonAriaLabel since that isn't necessary for the purposes of this test.